browser/lib/findNoteTitle.js

```diff
@@ -0,0 +1,32 @@
+export function find (value) {
```

Comment:
Because I don't like 
repetition like `findNoteTitle.findNoteTitle()`. What do you think `noteUtils.findNoteTitle()`?

The reason why I didn't choose that name is because I add only 1 method this time. I thought don't need to change the name to `noteUtils` unless need to add a lotta methods.